Assets/_Functionalities/AssetBundles/Scripts/AssetBundleLoader.cs

```diff
@@ -98,6 +98,7 @@ private static bool IsPrefab(GameObject gameObject)
 
         private IEnumerator GetAssetBundle(string path, UnityAction<AssetBundle> callBack)
         {
+#if UNITY_WEBGL && !UNITY_EDITOR
```

Comment:
Dit is niet wat je bedoelde: want nu wordt alleen in WebGL de onderstaande code uitgevoerd terwijl je een afslag voor alleen de editor hier bedoeld was. Dit faalt nu in een build anders dan WebGL

Reply:
dit klopt wel toch? als het platform webgl is en je zit niet in de editor: download de bundle via webrequest. anders gebruik je het file open systeem